akaze/src/image.rs

```diff
@@ -84,6 +104,46 @@ impl GrayFloatImage {
                     Luma([float_image[(x, y)].to_luma()[0]])
                 })
             }
+            DynamicImage::ImageRgb8(rgb_image) => {
```

Comment:
I believe that this and the followings are not necessary, because the `grayscale()` call at line 48 cannot return these. I think that only `ImageLumaA8` and `ImageLumaA16` are required.